syntax = "proto3";
package dbclient;




import "database_delivery_sdk/model/easy_command/task_callback.proto";
import "database_delivery_sdk/model/easy_command/target_log.proto";

/*
CreateClientCallback返回
*/
message CreateClientCallbackResponse {
    /*
    回调状态
    */
    string status = 1 ;
}

/*
CreateClientCallbackApi返回
*/
message CreateClientCallbackResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    dbclient.CreateClientCallbackResponse data = 4 ;
}

